src/meta/processors/admin/ListSnapshotsProcessor.cpp

```diff
@@ -32,6 +31,7 @@ void ListSnapshotsProcessor::process(const cpp2::ListSnapshotsReq&) {
         snapshots.emplace_back(std::move(snapshot));
         iter->next();
     }
+    resp_.set_code(to(ret));
```

Comment:
Why ?  Don't need an error code when the running fail?

Reply:
Should move it after prefix scan